WordPress/Classes/Models/AllTimeStatsRecordValue+CoreDataClass.swift

```diff
@@ -8,3 +8,27 @@ public class AllTimeStatsRecordValue: StatsRecordValue {
         try singleEntryTypeValidation()
     }
 }
+
+extension StatsAllTimesInsight: StatsRecordValueConvertible {
+    func statsRecordValue(in context: NSManagedObjectContext) -> StatsRecordValue {
+        let value = AllTimeStatsRecordValue(context: context)
+
+        value.postsCount = Int64(self.postsCount)
+        value.viewsCount = Int64(self.viewsCount)
+        value.visitorsCount = Int64(self.visitorsCount)
+        value.bestViewsPerDayCount = Int64(self.bestViewsPerDayCount)
+        value.bestViewsDay = self.bestViewsDay as NSDate
+
+        return value
+    }
+
+    init(statsRecordValue: StatsRecordValue) {
+        // We won't be needing those until later. I added them to protocol to show the intended design
+        // but it doesn't make sense to implement it yet.
+        fatalError()
```

Comment:
I'm agree with the notion of deferring the implementation here. 

Would you consider transposing some of the documentation above to the message sent to `fatalError(_:file:line:)` I'm thinking about it in the way that `init(coder:)` passes `"init(coder:) has not been implemented"`

It might lend some context on the off chance we encounter a fatal error subsequently.